app/src/pages/inside/common/durationBlock/durationBlock.scss

```diff
@@ -59,3 +59,10 @@
     line-height: 18px;
   }
 }
+
+.tooltip-duration {
+  padding: 24px 32px;
+  background: rgba(34, 34, 34, 0.91);
+  box-shadow: 0px 1px 3px rgba(55, 67, 98, 0.1);
```

Comment:
I think it is better to use here a rgba function. It enables you to use hex colors.
Please, use colors from our colors library.

Reply:
fixed ✅